File: week 1/pset1/mario (less)/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    //Asking for the height
    int h;
    do
    {
        h = get_int("Height = ");
    }
    while (h < 1 || h > 8); //Cheking if the input is vaild or not

    //Building the pyramid
    for (int i = 1; i < h + 1; i++)
    {
        for (int d = 0; d < h - i; d++)
        {
            printf(" ");
        }
        for (int j = 0; j < i; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: week 6/pset6/mario (more)/mario.py
from cs50 import get_int

# prompt the user for input
while True:
    height = get_int("height = ")
    if height < 9 and height > 0:
        break

# Build the pyramid
for i in range(1, height + 1):
    for d in range(0, height - i):
        print(" ", end="")
    for j in range(i):
        print("#", end="")
    print("  ", end="")
    for j in range(i):
        print("#", end="")
    print("")
